### DISCUSSION TITLE
How to complately remove index?

### DISCUSSION THREAD

**Discussion by roamerop** [JanusGraph/janusgraph#2500] [Q&A] [answered]
From the doc say,  [link](https://docs.janusgraph.org/index-management/index-removal/#example-for-mapreduce)
When i remove the index, it still exists with disable status.  So, how can i complately remove index if i want to create the index again. 
```
// Index still shows up in management interface as DISABLED -- this is normal
m = graph.openManagement()
idx = m.getGraphIndex('name')
idx.getIndexStatus(m.getPropertyKey('name'))
m.rollback()
```

**Comment by rngcntr** [comment, marked as answer]
Solved by #3362. Docs will be updated as the new version gets releases.
